crud.go

```diff
@@ -80,6 +80,58 @@ func (cc *CrudComponent) Get(ctx context.Context, opts *GetOptions) (*GetResult,
 		})
 }
 
+type GetReplicaOptions struct {
+	Key            []byte
+	ScopeName      string
+	CollectionName string
+	ReplicaIdx     uint32 // TODO(chvck): Check if this should be an uint32
```

Comment:
So it should indeed be uint32? In gocbcore it's int.

Reply:
It's an int in gocbcore because we support using negative numbers to target servers by index, but we have a better mechanism in gocbcorex to do this, by targeting a specific server via its endpoint name.